src/subscription_manager/managercli.py

```diff
@@ -2907,7 +2907,15 @@ def _do_command(self):
 
         syspurpose_cache = inj.require(inj.SYSTEMPURPOSE_COMPLIANCE_STATUS_CACHE)
         syspurpose_cache.load_status(self.cp, self.identity.uuid, on_date)
-        print(_("System Purpose Status: %s\n") % syspurpose_cache.get_overall_status())
+        print(_("System Purpose Status: %s") % syspurpose_cache.get_overall_status())
+
+        syspurpose_status_code = syspurpose_cache.get_overall_status_code()
+        if syspurpose_status_code != 'valid':
```

Comment:
I like more current implementation for two reasons:
1. When some other state will be added in the future and there will be some reasons for this state, then these reasons will be printed.
2. When the state is e.g. `disabled` and there are no reasons, then nothing is printed.

Reply:
Then I would suggest to change it to simply be `if syspurpose_status_code != 'matched'`, since `valid` is no longer used for syspurpose states in the latest candlepins in both hosted & standalone.